www/updateUes.php

```diff
@@ -6,10 +6,31 @@
  * Script for updating UES in separate thread.
  */
 
+use SimpleSAML\Configuration;
 use SimpleSAML\Logger;
 use SimpleSAML\Module\perun\Adapter;
 use SimpleSAML\Module\perun\ChallengeManager;
 
+const CLASS_PREFIX = 'perun/www/updateUes.php: ';
+const CONFIG_FILE_NAME = 'module_perun.php';
+const CONFIG_SECTION = 'updateUes';
+
+const USER_IDENTIFIERS = 'userIdentifiers';
+const SOURCE_IDP_ENTITY_ID = 'sourceIdPEntityID';
+
+const NAME = 'name';
+const FRIENDLY_NAME = 'friendlyName';
+const ID = 'id';
+const VALUE = 'value';
+const NAMESPACE_KEY = 'namespace';
+
+const TYPE = 'type';
+const STRING_TYPE = 'String';
+const INTEGER_TYPE = 'Integer';
+const BOOLEAN_TYPE = 'Boolean';
+const ARRAY_TYPE = 'Array';
+const MAP_TYPE = 'Map';
+
 $adapter = Adapter::getInstance(Adapter::RPC);
```

Comment:
I believe this will fail if the config file does not exist. check the `getInstance` method in the RPC adapter.

Reply:
This line does not use (and does not need) config at all.

Reply:
Overall, it is not an issue. Without the Adapter, we cannot continue. I just wanted to point it out to you.